Themer/build.gradle.kts

```
@@ -1,8 +1,11 @@
-version = "3.6.3"
+version = "3.6.4"
 description = "Apply custom themes to your Discord"
 
 aliucord.changelog.set(
     """
+    # 3.6.4
+    * Fix color theming for some elements like user profile header and chatbox
```

You made a terrible terrible typo

```suggestion
    * Fix colour theming for some elements like user profile header and chatbox
```

---

oh no that was close